lib/tasks/tasks.rake

```diff
@@ -48,6 +48,14 @@ namespace :tasks do
     AppInstallation.sync_all
   end
 
+  desc "Update all users to display comments in Octobox"
+  task display_comments: :environment do
+    User.all.each do |user|
```

Comment:
You can do this with a single sql update: `User.update_all(display_comments: true)`